tests/utils/dockerized.py

```diff
@@ -0,0 +1,44 @@
+import docker
+import pytest
+from pathlib2 import Path
+
+
+def dockerized_case(builder_cls, file_path):
+    """Decorator that makes decorated test case run inside docker container."""
+    def dockerized_real(f):
+        root = Path(__file__).parent.parent.parent
+        rel_path = Path("agent_source") / Path(file_path).relative_to(root)
+
+        command = "python -u -m pytest {0}::{1} -s --no-dockerize".format(
+            rel_path,
+            f.__name__
+        )
+
+        def wrapper(*args, **kwargs):
+            no_dockerize = pytest.config.getoption("--no-dockerize")
```

Comment:
Just a heads up - https://docs.pytest.org/en/latest/deprecations.html#pytest-config-global

Reply:
Oh, didn't know about that... thanks)